README.md

```diff
@@ -262,4 +262,8 @@ and run that simulator as usual:
         $ cd  builds/RV32CIUxCHERI_<CPU>_bluesim
         $ make compile simulator test isa_tests
 
+#### Clone submodules
```

Comment:
This could actually perhaps be more prominent, e.g. just after the intro paragraph of "What you can build and run if you have Bluespec's bsc compiler"